packages/react/jest.setup.ts

```diff
@@ -1 +1,5 @@
 import '@testing-library/jest-dom';
+import { TextDecoder, TextEncoder } from 'util';
+
+global.TextEncoder = TextEncoder;
+global.TextDecoder = TextDecoder;
```

Comment:
This is necessary in order to use isomorphic dompurify: https://github.com/kkomelin/isomorphic-dompurify/issues/91#issuecomment-1012645198

Reply:
please put this comment in the actual jest.setup file